modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -67,3 +67,63 @@ console.log(comments.getEventLog(
   { after: '2022-02-22T12:34:56Z' }  // ISO-8601 standard: YYYY-MM-DDThh:mm:ssZ
 ));
 ----
+
+[[editorannotator]]
+=== `+editor.annotator+`: `+'tinycomments'+` annotator
```

Comment:
The other API documented on this page is `getEventLog()`, and that is the string used for the sub-head that start’s the docs for that API.

I’m not arguing against this second sub-head, per se. But is there an edit that would make the new sub-head present more in line with the extant sub-head?

For example, is 

```suggestion
=== editor.annotator()
```
An acceptable alternative?

Reply:
The annotator API is already documented elsewhere, and is part of the editor. This `tinycomment` annotator is an "annotator" created for the comments plugin, not a separate API. I didn't see a need to create a separate heading for "Annotators", nor updating the filename, as this annotator can fit under the API heading, just as an implementation of another API.

Reply:
BTW I'm going to revert your changes as you have brought a commit from the staging branch, I'll just amend my previous commit with your changes.

Reply:
I hope my changes now have made it clearer